### ISSUE TITLE
Nothing happends after selecting the CC component 

### ISSUE THREAD

**Issue by sunel** [magesuite/content-constructor-admin#6]
After selecting the component from component section page nothing happens, No errors in the console.

**Comment by diwipl** [comment]
Hi, 
Can you record your browser along with developer console ?

**Comment by maquowicz** [comment]
Hi. I must say I'm impressed with your work.
Thank you for sharing.

Same issue here. 

Magento version: 2.4.5, php 8.1, nginx, firefox / chromium, *.test domain.
Had a node (~10) version warning on theme yarn install, ignored.
Added empty child theme with parent "Creativestyle/theme-creativeshop".
Custom code deregistered. All grinders disabled (bundling, merge, minification, signing).
Full generated cleanup and c:c. Nothing abnormal in logs.
Most of m2 crapware removed (can attach replacements), msi present, composer require:
```
    "require": {
        "ext-fileinfo": "*",
        "ext-json": "*",
        "ext-mbstring": "*",
        "ext-soap": "*",
        "bluepayment-plugin/module-bluepayment": "^2.21",
        "creativestyle/magesuite-elasticsuite-virtual-category-indexer": "^1.1.10",
        "creativestyle/magesuite-magepack": "^1.1",
        "creativestyle/magesuite": "~v12.0.0",
        "integer-net/magento2-global-custom-layout": "^1.1",
        "magefan/module-blog": "^2.2.8",
        "magento/composer-root-update-plugin": "~2.0",
        "magento/product-community-edition": "2.4.5",
        "mageplaza/magento-2-polish-language-pack": "dev-master",
        "mageplaza/module-smtp": "^4.6",
        "sparsh/magento-2-faq-extension": "~1.2",
    },
```

As far as I was able to look around. I'm missing viewport size selector on top of CMS Page > Content Tab.
![Screenshot from 2022-12-08 19-20-15](https://user-images.githubusercontent.com/10026557/206535265-59c331e4-267c-424e-be40-0536e34ca135.png)
Once any component selected console throws 
`Uncaught TypeError: this.ccConfig.sections is undefined` 
or in chrome:
`Uncaught TypeError: Cannot read properties of undefined (reading 'defaults') `
![Screenshot from 2022-12-08 19-31-18](https://user-images.githubusercontent.com/10026557/206538202-7df0f6fc-9409-496f-9224-369655435c0b.png)

Thanks for your time.

**Comment by diwipl** [comment]
Can you show a print screen of themes configuration in the admin panel?

The error suggests to me that theme creativeshop is not selected as the main theme there.

**Comment by maquowicz** [comment]
Sure. I'm using a child which has Creativestyle/theme-creativeshop as parent. 
But also tried switching to creativeshop - no luck. 
Will play around with it a bit more in couple. Need to afk atm.
![Screenshot from 2022-12-08 19-51-02](https://user-images.githubusercontent.com/10026557/206542332-e99501e6-4fdf-49be-b7d8-a83c92d67571.png)
![Screenshot from 2022-12-08 19-50-56](https://user-images.githubusercontent.com/10026557/206542373-b3be7bbd-c856-47d5-8733-a98cb9f06073.png)

**Comment by diwipl** [comment]
The design configuration screenshot looks really weird, it should have at least 3 rows by default, not only one.
<img width="1558" alt="image" src="https://user-images.githubusercontent.com/3406741/206543340-4fe9e6d5-0793-4dc0-a0eb-1521c757a343.png">

**Comment by maquowicz** [comment]
Yes, that's probably because I have Single Store Mode On.
![Screenshot from 2022-12-08 22-16-23](https://user-images.githubusercontent.com/10026557/206569141-4edf1d29-5af2-438f-94b0-fbc600898fdd.png)

**Comment by diwipl** [comment]
We have never tested it with Enable Single-Store mode so it might not work with such a setting.

**Comment by maquowicz** [comment]
Yes that is confirmed. Disabling Single-Store Mode and configuring "Theme Creativeshop" as global under Content > Design > Configuration solved my issue. All looking good now. 

Thank you Krzysztof!
